python/cudf/cudf/io/csv.py

```diff
@@ -77,7 +77,7 @@ def read_csv(
         dayfirst=dayfirst,
         compression=compression,
```

Comment:
Couldn't `compression` here also technically be a `np.str_`? I think we should handle the conversion of the input to `str` within the `read_csv` function rather than expect its callers to do that.

Reply:
Yep, good catch actually..handled it aswell